### PR TITLE
Add yanggangtony to istio member list

### PR DESCRIPTION
**Pull request by yanggangtony**
## I want to join the Istio org!

- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://slack.istio.io)
- [x] I have written my company name or "Individual", if not affliated with a company, here: ___ DaoCloud_______
- [x] I have provided a link to at least one PR that I have successfully pushed to one of the Istio repos, here: _____istio_____

I am a k8s and k8s-sigs member , and i like istio too. And want to contribute more in the future.

The pr i commit for istio is the link
[istio prs](https://github.com/istio/istio/pulls?q=is%3Apr+author%3Ayanggangtony+is%3Aclosed)

https://github.com/istio/istio/pull/42575

### CODE REVIEW

**Review by zirain — DISMISSED** [review]
Welcome!

File: org/members.yaml [hunk 1]
```diff
@@ -416,6 +416,7 @@ members:
 - Xunzhuo
 - yangminzhu
 - yankay
+- yanggangtony
```
Comment (zirain) [inline org/members.yaml:419]: Please sort in alphabetical order

Comment (yanggangtony) [reply org/members.yaml:419]: thanks.
Rebased.

**Review by zirain — APPROVED** [review]

**Comment by yanggangtony** [comment]
Thanks very much..

**Pull request merged**